airflow/providers/amazon/aws/hooks/s3.py

```diff
@@ -1274,7 +1274,7 @@ def copy_object(
 
         copy_source = {"Bucket": source_bucket_name, "Key": source_bucket_key, "VersionId": source_version_id}
         response = self.get_conn().copy_object(
-            Bucket=dest_bucket_name, Key=dest_bucket_key, CopySource=copy_source, ACL=acl_policy
+            Bucket=dest_bucket_name, Key=dest_bucket_key, CopySource=copy_source, ACL=acl_policy, **kwargs
```

Comment:
It would not work, because `S3Hook.copy_object` method doesn't implements `kwargs` argument

https://github.com/apache/airflow/blob/aed3c922402121c64264654f8dd77dbfc0168cbb/airflow/providers/amazon/aws/hooks/s3.py#L1228-L1236

Reply:
I have already added it